Python/initconfig.c

```diff
@@ -80,6 +80,7 @@ static const char usage_5[] =
 "PYTHONHOME   : alternate <prefix> directory (or <prefix>%lc<exec_prefix>).\n"
 "               The default module search path uses %s.\n"
 "PYTHONCASEOK : ignore case in 'import' statements (Windows).\n"
+"PYTHONUTF8: if this variable set to 1, it enables the UTF-8 mode.\n"
```

Comment:
I suggest to make the doc shorter:

```suggestion
"PYTHONUTF8: if set to 1, enable the UTF-8 mode.\n"
```

https://docs.python.org/dev/using/cmdline.html#envvar-PYTHONUTF8

But your change is correct as well :-)